### PR TITLE
add grant mention

### PR DESCRIPTION
**Pull request by bellesea**
would appreciate wording (& other) suggestions :) thank you!

### CODE REVIEW

**Review by maxwofford — APPROVED** [review]
This looks great! The top area feels kinda lengthy, but the announcement div is in just the right spot. The contrast on the announcement makes it kinda hard to see that it's a button.

**Review by sampoder — COMMENTED** [review]
It might be worth adding something to the TLDR or (IMO what I prefer) moving the announcement to the top

**Comment by bellesea** [comment]
changes: moved the announcement up + added shadow + added note to tldr 

(i think moving it up allows for more people to see it but could take away from the point of the page...)

File: pages/content/sponsorship.mdx [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+    mx: '0',
+    mt: '0',
+    background: '#f9fafc !important',
+    boxShadow: '0 4px 8px rgb(0 0 0 / 13%)'
+}}
+/>
+
+_(To summarize: we’re a donor-supported nonprofit so outside of planned grant programs, we don't provide financial support to hackathons/other nonprofits. However, there are all kinds of ways we’d love to help out your project, so read on.)_
+
+[Hack Club](https://hackclub.com/) is an independent nonprofit, supported by donors, with a responsibility to our supporters to spend our budget directly on our core programs. Therefore, as much as we’d like to, **we’re not in a position to provide financial support to hackathons/other nonprofits**. Although, we might run official grant programs, like a [$500 grant for all IRL high school hackathons](https://hackclub.com/hackathons/grant). We wish you the best of luck in your sponsorship search—we recommend [Megan Cui’s “Meginar”](https://youtu.be/tOmXzA4reTY) and [Lachlan Campbell’s Flagship talk](https://notebook.lachlanjc.com/2020-01-19_how_to_start_your_first_hackathon/) if you’re looking for advice.
```
Comment (maxwofford) [inline pages/content/sponsorship.mdx]:
```suggestion
[Hack Club](https://hackclub.com/) is an independent nonprofit, supported by donors, with a responsibility to our supporters to spend our budget directly on our core programs. Therefore, as much as we’d like to, **we’re not in a position to provide financial support to hackathons/other nonprofits**. We occasionally run grant programs, like the [$500 grant for IRL high school hackathons](https://hackclub.com/hackathons/grant). We wish you the best of luck in your sponsorship search—we recommend [Megan Cui’s “Meginar”](https://youtu.be/tOmXzA4reTY) and [Lachlan Campbell’s Flagship talk](https://notebook.lachlanjc.com/2020-01-19_how_to_start_your_first_hackathon/) if you’re looking for advice.
```

**Review by maxwofford — APPROVED** [review]
Added a slight change to wording. I'm cool with this merging.

**Comment by sampoder** [comment]
Thank you all!

**Pull request merged**